src/db/table/schemaSurvey/chainNodeDef.ts

```diff
@@ -0,0 +1,15 @@
+import { TableSchemaSurvey } from './tableSchemaSurvey'
+import { Column, ColumnType } from '../../column'
+
+export class TableChainNodeDef extends TableSchemaSurvey {
```

Comment:
can TableChainNodeDef and TableChainNodeDefAggregate extent a common class?

Reply:
why? what do they have in common?

Reply:
I would leave Table classes extends only the TableSchema ones, otherwise we risk to create a huge hierarchy of tables...it can be difficult to read the code

Reply:
ok